### DISCUSSION TITLE
The connection disappears and stops working

### DISCUSSION THREAD

**Discussion by bit-config** [nextcloud/all-in-one#3327] [Questions] [answered]
Doctrine\DBAL\Exception: Failed to connect to the database: An exception occurred in the driver: SQLSTATE[08006] [7] connection to server at "nextcloud-aio-database" (172.18.0.4), port 5432 failed: FATAL:  remaining connection slots are reserved for non-replication superuser connections in /var/www/html/lib/private/DB/Connection.php:140

After such a mistake, through repeating it. Everything stops working at all
Only a full reboot helps

**Comment by szaimen** [comment, marked as answer]
Hi, please disable collabora until a new release of the richcoduments app is published which will fix the issue.

**Reply by bit-config** [reply]
And there is no way to roll back the version?. I just have a job tied up on online documents

**Reply by mikayzo** [reply]
i dont even have collabora and still get the same issue

**Comment by juanyunis** [comment]
> i dont even have collabora and still get the same issue
Hi, were you able to fix it?
